psi4/src/read_options.cc

```diff
@@ -1439,6 +1439,12 @@ int read_options(const std::string &name, Options &options, bool suppress_printi
         /*- When using |scf__stability_analysis| ``FOLLOW``, maximum number of orbital optimization attempts
             to make the wavefunction stable. !expert -*/
         options.add_int("MAX_ATTEMPTS", 1);
+        /*- Which method to use to accelerate initial SCF convergence?-*/
+        options.add_str("INITIAL_SCF_ACCELERATOR", "ADIIS", "NONE EDIIS ADIIS");
+        /*- When the SCF error hits this number, start the linear blend of the initial SCF method out of SCF steps. -*/
```

Comment:
```suggestion
        /*- Value of SCF error at which to start mixing in DIIS with the initial SCF accelerator -*/
```

Reply:
Reworded my own way, though I like the idea of "at which" here.